bin/fluentd-windows-cmd

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'win32/event'

```

Comment:
On non-Windows environment, `win32/event` shouldn't be installed as a dependency and loaded.
Should we use `Fluent.windows?` for guarding to require on non-Windows environment?

Reply:
Or, just raise LoadError here is enough?

Reply:
Ditto